x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -18,7 +18,6 @@ export function createJestConfig({ kibanaDirectory, rootDir, xPackKibanaDirector
       '^src/core/(.*)': `${kibanaDirectory}/src/core/$1`,
       '^src/legacy/(.*)': `${kibanaDirectory}/src/legacy/$1`,
       '^src/plugins/(.*)': `${kibanaDirectory}/src/plugins/$1`,
-      '^plugins/([^/.]*)(.*)': `${kibanaDirectory}/src/legacy/core_plugins/$1/public$2`,
```

Comment:
If you paste these 2 files ([file on master](https://raw.githubusercontent.com/elastic/kibana/master/x-pack/dev-tools/jest/create_jest_config.js), [file on this commit](https://raw.githubusercontent.com/elastic/kibana/c55baa138248ab02b5bfabdf87f88091200a24b9/x-pack/dev-tools/jest/create_jest_config.js)) into [diffchecker.com](https://www.diffchecker.com/diff), they are identical. no idea what git is talking about